src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -594,6 +594,13 @@ extension SecureStore: SRSProviding {
         get { kvStore["otpSrsAuthorizationDate"] as Date? }
         set { kvStore["otpSrsAuthorizationDate"] = newValue }
     }
+	
+	#if !RELEASE
+	var isSRSPrechecksEnabled: Bool {
```

Comment:
To follow the pattern above 

```suggestion
	var isSrsPrechecksEnabled: Bool {
```